test/spec/functions.ts

```diff
@@ -87,16 +78,7 @@ function eth_fast_aggregate_verify(input: {pubkeys: string[]; message: string; s
     return true;
   }
```

Comment:
I really dont like this hack here because this was the reason this PR was created.  To remove all the code that was like this.  I am not sure how to remove this though.  The `bls` and `phase0` specs call for this condition to return `false` so that is what is returned from the native layer.  For `altair` this was changed to return `true`.  Without passing the fork, which this library has no view of, there is no way to get both false and true to return from identical test cases. 

So this hack had to stay....